CITATION.cff

```diff
@@ -0,0 +1,47 @@
+cff-version: 1.2.0
+title: "GLASS: Generator for Large Scale Structure"
+message: >-
+  If you use this software, please cite it using the metadata from this file.
+preferred-citation:
```

Comment:
This is going to get confusing. Could you move the `preferred-citation` as the last entry in the YAML? Currently there is a mismatch in the authors

Reply:
Like would this work

```yaml
cff-version: 1.2.0
title: "GLASS: Generator for Large Scale Structure"
message: >-
  If you use this software, please cite it using the metadata from this file.
identifiers:
  - type: doi
    value: 10.21105/astro.2302.01942
keywords:
  - Astrophysics
  - Cosmology
  - Nongalactic Astrophysics
license: MIT
repository-code: "https://github.com/glass-dev/glass"
url: "https://glass.readthedocs.io"
preferred-citation:
  authors:
    - family-names: Nicolas
      given-names: Tessore
      orcid: https://orcid.org/0000-0002-9696-7931
  date-published: 2023-03
  doi: 10.21105/astro.2302.01942
  eid: 11
  eprint: 2302.01942
  journal: The Open Journal of Astrophysics
  issn: 2475-9066
  issue: 84
  pages: 11
  primaryClass: astro-ph.CO
  title: "GLASS: Generator for Large Scale Structure"
  type: article
  volume: 6
```

Reply:
Do we just want this or the rest of the existing keys and values go below this?

Reply:
I think for now keep what you have and move `preferred-citation` ot the bottom. Makes it easier to read for the review.